images/salt-master/Dockerfile

```diff
@@ -26,6 +26,7 @@ gpgkey=https://repo.saltstack.com/yum/redhat/\$releasever/\$basearch/archive/%s/
  && yum install -y python2-kubernetes salt-master salt-api salt-ssh openssh-clients \
  && yum install -y python-pip \
  && yum clean all \
+ && pip install -U pip \
```

Comment:
Maybe I should freeze the pip version to the latest version available now (20.0.2), rather than doing that.

Reply:
Yup, better freeze the version.

Reply:
Maybe, though probably much less critical than freezing `etcd3` just below, since we rely on it at runtime.